example-docker.yml

```
@@ -0,0 +1,28 @@
+FROM eclipse-temurin:17-jdk-jammy as build
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)